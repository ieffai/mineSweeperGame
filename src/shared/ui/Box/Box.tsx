import { default as BaseBox, type BoxProps as BaseBoxProps } from '@mui/material/Box';

export type BoxProps = BaseBoxProps;
export const Box = BaseBox;

export default Box;
