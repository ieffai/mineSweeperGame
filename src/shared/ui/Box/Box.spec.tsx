import { render } from '@testing-library/react';

import Box from './Box';

describe('Box', () => {
  it('should render successfully', () => {
    const { asFragment } = render(<Box />);
    expect(asFragment()).toMatchSnapshot();
  });
});
