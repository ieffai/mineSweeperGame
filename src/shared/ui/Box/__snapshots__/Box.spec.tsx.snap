// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Box > should render successfully 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  />
</DocumentFragment>
`;
